installation/cmd/run-from-kyma.sh

```diff
@@ -17,4 +17,4 @@ mv -f ${OVERRIDES_MINIKUBE_PATH}.bak $OVERRIDES_MINIKUBE_PATH
 
 # TODO: Remove it after next CLI release
 echo "Adding Compass entries to /etc/hosts...\n"
-sudo sh -c 'echo "\n$(minikube ip) compass-gateway.kyma.local" >> /etc/hosts'
+sudo sh -c 'echo "\n$(minikube ip) compass-gateway.kyma.local compass.kyma.local" >> /etc/hosts'
```

Comment:
You need to add `compass-mf.kyma.local` as well

Reply:
After modification, please also copy the fragment from line 18 to 20 to `run.sh` script. It should be added there as well.